src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/RememberMeFactory.php

```diff
@@ -104,6 +104,10 @@ public function createAuthenticator(ContainerBuilder $container, string $firewal
             $loader->load('security_authenticator_remember_me.php');
         }
 
+        if ('auto' === $config['secure']) {
+            $config['secure'] = null;
+        }
```

Comment:
Here is the corresponding logic from the old implementation:
https://github.com/symfony/symfony/blob/4333d0bfe3342db2d6c42c6e57035aa2157e7cbb/src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/RememberMeFactory.php#L266-L268
It is needed for:
https://github.com/symfony/symfony/blob/4333d0bfe3342db2d6c42c6e57035aa2157e7cbb/src/Symfony/Component/Security/Http/RememberMe/AbstractRememberMeHandler.php#L123